themes/default/css/content.min.css

```diff
@@ -0,0 +1 @@
+/* use to remove skin from tinemce*/
```

Comment:
one comment line css?

Reply:
We need to clear the tine mce skin so that the sass can define the style instead...